drivers/storage/portworx/deployment.go

```diff
@@ -440,6 +482,27 @@ func (t *template) getSelectorRequirements() []v1.NodeSelectorRequirement {
 	return selectorRequirements
 }
 
+func (t *template) getCloudStorageArguments(cloudDeviceSpec cloudstorage.CloudDriveConfig) string {
+	devSpec := strings.Join(
+		[]string{
+			"type=" + cloudDeviceSpec.Type,
+			"size=" + strconv.FormatUint(cloudDeviceSpec.SizeInGiB, 10),
+			"iops=" + strconv.FormatUint(uint64(cloudDeviceSpec.IOPS), 10)},
```

Comment:
Question:
Will IOPS always be present in the output from cloudops?
For some providers or drive type, Portworx will ignore the iops param, right?

Reply:
Yes thats right. I think the operator should just pass on the parameters it gets from cloudops. The actual driver in Portworx can handle if it needs to ignore.